src/components/BooksList.js

```diff
@@ -0,0 +1,20 @@
+import BooksDetail from '../containers/BooksDetail';
+
+const BooksList = () => (
+  <div>
+    <table>
+      <tbody>
+        <tr>
+          <th>Id</th>
+          <th>Title</th>
+          <th>Category</th>
+        </tr>
+
+        <BooksDetail />
+
+      </tbody>
+    </table>
+  </div>
+);
+
+export default BooksList;
```

Comment:
- [ ] This should be placed in the `src/containers` directory.
- [ ] it should display books list in html table using `Book` component
- [ ] It should be connected to the redux store and fetch books from the state
 screenshot from pathwright 👇🏾 
![Screen Shot 2021-02-04 at 9 01 04 PM](https://user-images.githubusercontent.com/48804258/106948534-1a284200-672c-11eb-9876-c2fd135c1250.png)